bin/parse-legacy-file

```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env bash
-echo $(
-  sed '1 s/^v//' "$1"
-)
+
+version=$(sed '1 s/^v//' "$1")
+
+if [[ "$version" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
I advise against using bash specific features and to keep the posix compatibility where possible, we had problems in the past with bashims and we like to keep the idea of eventually removing the bash dependency.

Try using `grep` or [expr](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/expr.html) and prefer the `*` regex modifier to the extended regex `+`. A neat posix version of this line would be:
```suggestion
if expr "$version" : '^[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*$'; then
```